### ISSUE TITLE
9.2.1 Decompiler background still white

### ISSUE THREAD

**Issue by aSmig** [zackelia/ghidra-dark#5]
Not sure if I did something wrong, just trying this for the first time. Fresh install of Ghidra 9.2.1 and ghidra-dark. Menus, buttons, scroll bars and project manager are dark. But Decompiler code browser is still bright.

**Comment by zackelia** [comment]
Just tried on a fresh installation of 9.2.1 and it seems fine for me, I also don't see anything indicating a change in the release notes. Can you try this and re-run the tool to see if it fixes it?
```
$ cp _code_browser.tcd.bak _code_browser.tcd
```

**Comment by aSmig** [comment]
I had been goofing around with the inverted color option after posting this issue so I pushed my install and .ghidra directories aside and am now unable to reproduce the issue I saw. Tried a couple times to go through the same steps I took the first time and have seen no issues each time. Thanks for the quick "fix" (IE, not screwing anything up in the first place)! Still dunno how I managed to break it in the first place. Shrug.

This looks great; my eyes thank you!
